sigmadsp/hardware/adau14xx.py

```diff
@@ -137,3 +142,22 @@ def adjust_volume(self, adjustment_db: float, address: int) -> float:
         )
 
         return linear_to_db(new_volume)
+
+    def safeload(self, address: int, data: bytes, count: int = 1):
+        """Write data to the chip using software safeload.
+
+        Args:
+            address (int): Address to write to
+            data (bytes): Data to write; multiple parameters should be concatenated
+            count (int): Number of 4 byte words to write
+        """
+        count = min(count, 5)
```

Comment:
```suggestion
        count = min(count, len(Adau14xx.SAFELOAD_DATA_REGISTERS))
```

However, shouldn't we actually raise an exception, if we wanted to write more than is possible with the safeload registers? E.g.:

```suggestion
        if count > len(Adau14xx.SAFELOAD_DATA_REGISTERS):
            raise IndexError(f"Cannot write {count * 4} bytes by means of software safeload, the maximum is {len(Adau14xx.SAFELOAD_DATA_REGISTERS) * 4} bytes.")
```